src/applications/beta-enrollment/routes.js

```diff
@@ -15,6 +16,14 @@ const routes = {
       path: 'personalization',
       component: createBetaEnrollmentButton(features.dashboard, '/dashboard'),
     },
+    {
+      path: 'all-claims',
+      component: createBetaEnrollmentButton(
+        features.allClaims,
+        '/disability-benefits/apply/form-526-all-claims',
```

Comment:
Also heads up: It looks like [this URL is changing](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15793), I think.

Reply:
Ah we should coordinate with that ticket—this is the URL that users will go to if they are not enrolled in beta and try to access the V2 form directly